pkg/cloudprovider/controller/nodecontroller.go

```diff
@@ -34,6 +34,30 @@ import (
 	"github.com/golang/glog"
 )
 
+const (
+	// The constant is used if sync_nodes_status=False. NodeController will not proactively
+	// sync node status in this case, but will monitor node status updated from kubelet. If
+	// it doesn't receive update for this amount of time, it will start posting node NotReady
+	// condition. The amount of time when NodeController start evicting pods is controlled
+	// via flag 'pod_eviction_timeout'.
+	// Note: be cautious when changing the constant, it must work with nodeStatusUpdateFrequency
+	// in kubelet. There are several constraints:
+	// 1. nodeStatusUpdateFrequency must be N times less than nodeMonitorGracePeriod, where
+	//    N means number of retries allowed for kubelet to post node status.
+	// 2. nodeMonitorGracePeriod can't be too large for user experience - larger value means
+	//    it takes longer for user to see update-to-date node status.
+	// 3. nodeStatusUpdateFrequency can't be too small, to allow kubelet prepare node status.
```

Comment:
Can you write this more precisely? For example "It is pointless to make nodeStatusUpdateFrequency be less than Kubelet's nodeStatusUpdateFrequency, since there will only be fresh values from Kubelet at an interval of nodeStatusUpdateFrequency."